alerta-covid/Gemfile

```diff
@@ -0,0 +1,67 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.2'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.3'
+# Use postgresql as the database for Active Record
+gem 'pg', '~> 1.1'
+# Use Puma as the app server
+gem 'puma', '~> 5.0'
+# Use SCSS for stylesheets
+gem 'sass-rails', '>= 6'
+# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
+gem 'webpacker', '~> 5.0'
+# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
+gem 'turbolinks', '~> 5'
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'jbuilder', '~> 2.7'
+# Use Redis adapter to run Action Cable in production
+# gem 'redis', '~> 4.0'
+# Use Active Model has_secure_password
+# gem 'bcrypt', '~> 3.1.7'
+
+gem 'cancancan'
+gem 'pundit'
+gem 'devise'
+gem 'will_paginate', '~> 3.1.0'
+gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
+gem 'figaro'
```

Comment:
Just curious. 
Is this gem really needed?

How do you plan to use it?

Sorry, it's just that I didn't see it before
I think we can make what it offers with vanilla ruby/rails

Reply:
Sorry :c, but we don't understand which gem we can make with ​​vanilla ruby/rails, are marked 4

Reply:
Figaro. I can see what the purpose of that gem is
> Figaro is inspired by the Twelve-Factor App methodology, which states:

But Rails is already 12-factor friendly, and I think Figaro it's not needed 😄

Reply:
Figaro we use it to use and config enviroment variables, so we dont have to upload our users and passwords in github